javascript-loops-practice/filter-out-strings.js

```diff
@@ -0,0 +1,7 @@
+/* exported filterOutStrings */
+function filterOutStrings(values) {
+  for (var i = values.length - 1; i >= 0; i--) {
+    if (typeof values[i] === 'string') { values.splice(i, 1); }
+  }
+  return values;
```

Comment:
This function should return a new array instead of deleting values from the old one.